CONTRIBUTING.md

```diff
@@ -126,7 +126,7 @@ The .strings files should still be in the original English – that’s expecte
 
 ### Supported devices:
 - iPhone 11 Pro Max, iOS 13.7
-- iPhone 12 Pro Max, iOS 14.4
+- iPhone 12 Pro Max, iOS 14.5
```

Comment:
Right now we're using `iOS 14.5` for iPhone 12 Pro Max in snapshot tests in fact.

Reply:
Please also add this change in `README.md`.